tests/ImageSharp.Tests/Metadata/Profiles/ICC/DataWriter/IccDataWriterPrimitivesTests.cs

```diff
@@ -42,7 +42,7 @@ public void WriteAsciiStringWithNullWritesEmpty()
         byte[] output = writer.GetData();
 
         Assert.Equal(0, count);
-        Assert.Equal([], output);
+        Assert.Equal(Array.Empty<byte>(), output);
```

Comment:
We shouldn’t need to do this. Might be something to do with breaking changes re span overloads though.